plugin/evict_leader/Makefile

```diff
@@ -0,0 +1,7 @@
+evictLeaderPlugin.so: *.go
```

Comment:
I prefer to move the whole `evict_leader` and `grant_leader` into a directory named such as `example` or `xxx_example`.